pmd-java/src/main/resources/category/java/codestyle.xml

```diff
@@ -36,71 +36,93 @@ by {% rule java/codestyle/ClassNamingConventions %}.
  [$strict='true']
  [starts-with(@Image, 'Abstract')]
 ]]>
-                </value>
-            </property>
-            <property name="strict" type="Boolean" value="true" description="Also flag classes, that are named Abstract, but are not abstract."/>
-        </properties>
-        <example>
-<![CDATA[
+        </value>
+      </property>
+      <property name="strict" type="Boolean" value="true"
+        description="Also flag classes, that are named Abstract, but are not abstract."/>
+    </properties>
+    <example>
+      <![CDATA[
 public abstract class Foo { // should be AbstractFoo
 }
 ]]>
-        </example>
-    </rule>
-
-    <rule name="AtLeastOneConstructor"
-          language="java"
-          since="1.04"
-          message="Each class should declare at least one constructor"
-          class="net.sourceforge.pmd.lang.java.rule.codestyle.AtLeastOneConstructorRule"
-          externalInfoUrl="${pmd.website.baseurl}/pmd_rules_java_codestyle.html#atleastoneconstructor">
-        <description>
-<![CDATA[
+    </example>
+  </rule>
+
+  <rule name="ArrayInitializationVerboseness"
+    language="java"
+    since="1.1"
```

Comment:
this should actually be 6.12.0, as it refers to the first version of PMD to support this rule

Reply:
I'll fix the version once I'm back from lunch. What do you mean by format change?

Reply:
You seem to have run an autoformatting of the XML, so there are lots of whitespace changes (indentation, line breaks, and so forth): https://github.com/pmd/pmd/pull/1647/commits/0a0209e70be303099bd9f450e542c53f7f64a0a0#diff-315d6ec2e1aa011e7ba88112a1b75f07